3.0.0 (#20)
MetaMask/superstruct: CHANGELOG.md

## [Unreleased]

## [3.0.0]

### Added

- Newly exports the following types: `AnyStruct`, `EnumSchema`, `InferStructTuple`, `IsExactMatch`, `IsMatch`, `IsRecord`, `IsTuple`, `ObjectSchema`, `OmitBy`, `Optionalize`, `PickBy`, `Simplify`, `UnionToIntersection` ([#18](https://github.com/MetaMask/superstruct/pull/18))

> ### Changed
> 
> - **BREAKING:** Expose separate build entry points and type declarations for CommonJS and ESM via package manifest `exports`. ([#18](https://github.com/MetaMask/superstruct/pull/18))
>   - When imported as a module, this package can now resolve as both CommonJS and ESM.

Author: ```suggestion
### Fixed

- **BREAKING:** Expose separate build entry points and type declarations for CommonJS and ESM via package manifest `exports` ([#18](https://github.com/MetaMask/superstruct/pull/18)).
```
